test/testdata

```diff
@@ -273,6 +273,8 @@ SP        GAMESSUK    GenericSPTest         basicGAMESS-UK8.0           dvb_sp_k
 SP        GAMESS      GenericHFSPTest       basicFirefly8.0             dvb_sp.out
 SP        GAMESS      GenericSPTest         basicGAMESS-US2017          dvb_sp.out
 SP        GAMESS      GenericSPTest         basicGAMESS-US2018          dvb_sp.out
+SP        GAMESSDAT   GenericSPTest         basicGAMESS-US2018          water_ccd.dat
```

Comment:
These aren't the single-point files; they should be `dvb_sp.dat`.

Reply:
Fixed